src/components/Results/utils.ts

```diff
@@ -101,21 +100,24 @@ const getType = (study: ResearchStudy): string => {
   }
 };
 
-const getClosestFacilities = (locations: Location[], zipcode: string, numOfFacilities = 1): ContactProps[] => {
-  // For now, this only supports getting the first facility.
+const getClosestFacilities = (locations: Location[], zipcode: string, numOfFacilities = 5): ContactProps[] => {
   const origin = getZipcodeCoordinates(zipcode);
-  const locationsWithCoordinates = getCoordinatesForLocations(locations);
-  const closest = getCoordinatesOfClosestLocation(origin, locationsWithCoordinates);
-  const distance = getDistanceBetweenPoints(origin, closest);
-
-  // Again, for now, just get the closest facility
-  const location = closest && locationsWithCoordinates.find(coordinatesAreEqual(closest));
-  return [
-    {
+  const locationCoordinates = getCoordinatesForLocations(locations);
+  const closest = locationCoordinates
+    .sort((first, second) => {
+      const firstToOrigin = getDistanceBetweenPoints(origin, getLocationCoordinates(first));
```

Comment:
Does it make sense for these function calls to happen in the comparison function? Because `.sort` is a comparative sort (probably `qsort`) so we'd be doing the distance calculation every time (IE, probably `nlogn` times); the alternative is apply the distance calculation to the whole array and then sort. That being said I don't actually know if the distance calculation is extensive (so this might be a moot point). 

Additionally, you wouldn't need to call the `getDistanceBetweenPoints` function again on line 115 since you've already calculated it.